ztp/resource-generator/src/watcher.py

```diff
@@ -59,27 +59,25 @@ class PolicyGenWrapper(Logger):
     def __init__(self, paths: list):
         try:
             folders = [{'input': paths[0], 'output': paths[1]}]
-            src = '/usr/src/hook/cnf-features-deploy'
-            dest = '/tmp/cnf-features-deploy'
+            src = '/usr/src/hook/ztp'
+            dest = '/tmp/ztp'
+            shutil.rmtree(dest, ignore_errors=True)
             shutil.copytree(src, dest)
-            cwd = '/tmp/cnf-features-deploy/ztp/ztp-policy-generator'
-            command = 'kustomize build --enable-alpha-plugins'
-            oneliner_file = 'policyGenerator.yaml'
+            cwd = os.path.join(
+                '/tmp/ztp/ztp-policy-generator',
+                'kustomize/plugin/policyGenerator/v1/policygenerator/')
+            command = [
```

Comment:
Just curious! How does this run without having kustomize on the container?

Reply:
So PolicyGenerator is just an executable that we happened to invoke as kustomize plugin before. But you can also run it standalone if you provide some dummy argument (see line 71) as argv[1], shifting by that all other arguments to their correct places